website/source/docs/builders/virtualbox-iso.html.md.erb

```diff
@@ -121,9 +121,9 @@ builder.
     ```
 
     A VirtualBox [VM
-    description](https://www.virtualbox.org/manual/ch08.html#idm3756) may
-    contain arbitrary strings; the GUI interprets HTML formatting. However, the
-    JSON format does not allow arbitrary newlines within a value. Add a
+    description](https://www.virtualbox.org/manual/ch09.html#vboxmanage-modifyvm-general)
```

Comment:
I think that we probably meant to reference the description in the export docs. 

```suggestion
    description](https://www.virtualbox.org/manual/ch09.html#9.12.1)
```

Reply:
That page doesn't seem to have a `#9.12.1` tag in it, so I used the section name instead. Should be the same effect as your suggestion.